src/host/main.cpp

```diff
@@ -142,6 +142,11 @@ int main(int argc, char** argv)
     json_log_path,
     "Path to file where the json logs will be written");
 
+  bool log_format_stdout_json = false;
```

Comment:
It seems odd to have separate options for "format as JSON in std out", and "format as JSON to this file". Can we remove the file option, and if someone wants this they can redirect stdout? And if we have a use case for multiple loggers, we can make these fully configurable from the CLI (ie - give a file _and_ a format type, and maybe a log level, and...)